libcontainer/utils/utils_unix.go

```diff
@@ -202,3 +202,8 @@ func ProcThreadSelf(subpath string) (string, ProcThreadSelfCloser) {
 	}
 	return threadSelf + subpath, runtime.UnlockOSThread
 }
+
+// ProcThreadSelfFd is shorthand for ProcThreadSelf(fmt.Sprintf("fd/%d", fd)).
```

Comment:
The comment is out of sync with the actual implementation

Reply:
It isn't the same as the actual implementation, but I thought the description being easily understandable was more important than matching the implementation exactly.

But I'll reword this to just explain what it does.